### PR TITLE
Reposition disabled courses

### PR DESCRIPTION
**Pull request by Huahan98**
## Description

Reposition disabled courses to the bottom of the course page

## Motivation and Context

This PR made change in `app/controllers/courses_controller.rb`. There was no specific logic written for how courses should be ordered, the order that the courses appear is simply the order of keys in the `listing` dictionary. Putting `disabled` as the last key solves the problem.

## How Has This Been Tested?

Tested locally, please refer to the image below

## Screenshots (if appropriate):
Screen Shot 2021-04-05 at 10 45 49 PM

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -426,13 +426,13 @@ def edit_course_params
...
   def categorize_courses_for_listing(courses)
     listing = {}
-    listing[:disabled] = []
 
     # temporal
     listing[:current] = []
     listing[:completed] = []
     listing[:upcoming] = []
 
+	listing[:disabled] = []
```
Comment (fanpu) [inline app/controllers/courses_controller.rb:435]: A tab was used here although the rest of the lines uses spacing so there's a spacing difference

**Review by fanpu — APPROVED** [review]
LGTM, can merge once the spacing issue is fixed!

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (Huahan98) [reply app/controllers/courses_controller.rb]: resolved!

**Review by fanpu — APPROVED** [review]
LGTM

**Pull request merged**